src/containers/CreateArticle/CreateArticlePage.js

```diff
@@ -0,0 +1,254 @@
+/* eslint-disable */
+import React, { Component, Fragment } from "react";
+import { connect } from "react-redux";
+import { Redirect } from "react-router-dom";
+import axios from "axios";
+import PropTypes from "prop-types";
+import createArticle from "../../actions/article";
+import ArticleTitle from "./ArticleTitle";
+import TextEditor from "./TextEditor";
+import ErrorPage from "../Error";
+import stripHtml from "../../utils/stripHtml";
+import styles from "./CreateArticle.module.scss";
+
+class CreateArticle extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      title: "Title",
+      body: "Share your thoughts...",
+      description: "",
+      error: {
+        title: "",
+        body: ""
+      },
+      message: ""
+    };
+  }
+
+  clearErrorMessages = () => {
+    this.setState({
+      error: {
+        title: "",
+        body: ""
+      }
+    });
+  };
+
+  titleChangeHandler = title => {
+    this.clearErrorMessages();
+    this.setState({
+      title
+    });
+  };
+
+  editorChangeHandler = body => {
+    this.clearErrorMessages();
+    this.setState({
+      body
+    });
+  };
+
+  titlePlaceholderFocusInHandler = () => {
+    const { title } = this.state;
+    if (title.trim() === "Title") {
+      this.setState({ title: "" });
+    }
+  };
+
+  titlePlaceholderFocusOutHandler = () => {
+    const { title } = this.state;
+    if (title.trim() === "") {
+      this.setState({
+        title: "Title"
+      });
+    }
+  };
+
+  bodyPlaceholderFocusInHandler = () => {
+    const { body } = this.state;
+    if (
+      body.trim() === "Share your thoughts..." ||
+      body.trim() === "<p>Share your thoughts...</p>"
+    ) {
+      this.setState({
+        body: ""
+      });
+    }
+  };
+
+  bodyPlaceholderFocusOutHandler = () => {
+    const { body } = this.state;
+    if (body.trim() === "") {
+      this.setState({
+        body: "Share your thoughts..."
+      });
+    }
+  };
+
+  imageUploadHandler = (blobInfo, success, failure) => {
+    try {
+      const formData = new FormData();
+      formData.append("file", blobInfo.blob());
+      formData.append("upload_preset", "asgjqcgx");
```

Comment:
Your cloudinary keys and url (on lines 94 and 97)should be better set on the .env file

Reply:
okay, will do that now